src/mixins/validatable-http-message.coffee

```diff
@@ -14,11 +16,11 @@ class Validatable
   #@return [Object] :headers {ValidationErrors}, :body {ValidationErrors}, :statusCode [Boolean]
   validate: () ->
     @validation = {}
-
+    @validation.version = '2'
     @lowercaseHeaders()
 
-    @validateHeaders() unless @headers == undefined
-    @validateBody() unless @body == undefined
+    @validateHeaders() unless @headers == undefined || @expected.headers == undefined
+    @validateBody() unless @body == undefined || @expected.body == undefined
     @validateStatusCode() unless @statusCode == undefined
```

Comment:
Why isnt' the same "or" as above used here too? (check for `expected.statusCode?`)